### PR TITLE
Add 2021.10.0 release note items in API and OIE API RNs

### PR DESCRIPTION
**Pull request by vanngo-okta**
## Description:
- **What's changed?** Add 2021.10.0 release note items in API and OIE API RNs
- **Is this PR related to a Monolith release?** Yes, 2021.10.0

### Resolves:

* [OKTA-432967](https://oktainc.atlassian.net/browse/OKTA-432967)

### CODE REVIEW

File: packages/@okta/vuepress-site/docs/release-notes/2021-okta-identity-engine/index.md [hunk 1]
```diff
@@ -4,6 +4,18 @@ title: Okta Identity Engine API Products Release Notes 2021
...
 
+## October
+
+### Monthly Release 2021.10.0
+
+| Change                                                                   | Expected in Preview Orgs |
+|--------------------------------------------------------------------------|--------------------------|
+| [Brands API option to remove sign-in page footer message](#brands-api-option-to-remove-sign-in-page-footer-message) | October 6, 2021 |
+
+#### Brands API option to remove sign-in page footer message
```
Comment (susanharper-okta) [inline packages/@okta/vuepress-site/docs/release-notes/2021-okta-identity-engine/index.md:15]:
```suggestion
#### New Brands API option to remove sign-in page footer message
```

just feels like it needed the word "new" to clarify if it was new or being taken away or what....

File: packages/@okta/vuepress-site/docs/release-notes/2021/index.md [hunk 2]
```diff
@@ -1,6 +1,39 @@
...
+## October
+
+### Monthly Release 2021.10.0
+
+| Change                                                                   | Expected in Preview Orgs |
+|--------------------------------------------------------------------------|--------------------------|
+| [AES-GCM encryption support for SAML assertions](#aes-gcm-encryption-support-for-saml-assertions) | October 6, 2021 |
+| [Authentication API returns UD User Profile locale is GA in Preview](#authentication-api-returns-ud-user-profile-locale-is-ga-in-preview) | October 6, 2021 |
+| [Native SSO support is GA in Preview](#native-sso-support-is-ga-in-preview) | October 6, 2021 |
+| [Brands API option to remove sign-in page footer message](#brands-api-option-to-remove-sign-in-page-footer-message) | October 6, 2021 |
```
Comment (susanharper-okta) [inline packages/@okta/vuepress-site/docs/release-notes/2021/index.md:14]:
```suggestion
| [New Brands API option to remove sign-in page footer message](#new-brands-api-option-to-remove-sign-in-page-footer-message) | October 6, 2021 |
```

File: packages/@okta/vuepress-site/docs/release-notes/2021/index.md [hunk 3]
```diff
@@ -1,6 +1,39 @@
...
+| [Bug fixed in 2021.10.0](#bug-fixed-in-2021-10-0)                          | October 6, 2021          |
+
+#### AES-GCM encryption support for SAML assertions
+
+To secure SAML assertions from attacks and to adopt a stronger security mechanism, Okta now supports AES128-GCM and AES256-GCM encryptions modes in addition to AES-128 and AES-256 for SAML applications. <!--OKTA-428056-->
+
+#### Authentication API returns UD User Profile locale is GA in Preview
+
+The response for the primary Authentication API includes the [User Profile's locale](/docs/reference/api/authn/#user-profile-object) value that is the same as the User Profile locale value in Universal Directory (UD). This behavior is now Generally Available in Preview. <!--OKTA-430697-->
```
Comment (susanharper-okta) [inline packages/@okta/vuepress-site/docs/release-notes/2021/index.md:23]: don't need to repeat the GA in Preview part in the content - it's in the title
```suggestion
The response for the primary Authentication API includes the [User Profile's locale](/docs/reference/api/authn/#user-profile-object) value that is the same as the User Profile locale value in Universal Directory (UD). <!--OKTA-430697-->
```

File: packages/@okta/vuepress-site/docs/release-notes/2021/index.md [hunk 4]
```diff
@@ -1,6 +1,39 @@
...
+
+#### Native SSO support is GA in Preview
+
+Native SSO support is now Generally Available in Preview. You can now configure your org to use the Native Single-Sign On (SSO) flow. The [Native SSO](/docs/guides/configure-native-sso/) feature allows you to protect native OpenID Connect applications and achieve SSO and Single Logout between native applications. <!--428053-->
```
Comment (susanharper-okta) [inline packages/@okta/vuepress-site/docs/release-notes/2021/index.md:27]: don't need to repeat the title in the content
```suggestion
You can now configure your org to use the Native Single-Sign On (SSO) flow. The [Native SSO](/docs/guides/configure-native-sso/) feature allows you to protect native OpenID Connect applications and achieve SSO and Single Logout between native applications. <!--428053-->
```

File: packages/@okta/vuepress-site/docs/release-notes/2021/index.md [hunk 5]
```diff
@@ -1,6 +1,39 @@
...
+
+#### Brands API option to remove sign-in page footer message
```
Comment (susanharper-okta) [inline packages/@okta/vuepress-site/docs/release-notes/2021/index.md:29]:
```suggestion
#### New Brands API option to remove sign-in page footer message
```

**Review by susanharper-okta — COMMENTED** [review]
couple of nits

File: packages/@okta/vuepress-site/docs/release-notes/2021/index.md (hunk 3 above)
Comment (vanngo-okta) [reply packages/@okta/vuepress-site/docs/release-notes/2021/index.md:23]: You're right, it is redundant. But I thought this our convention (https://oktawiki.atlassian.net/wiki/spaces/DOC/pages/2066710848/API+Release+Note+-+Blurb+Templates)? We need to say "GA in Preview', then repeat the full word... "Generally Available in Preview"?

File: packages/@okta/vuepress-site/docs/release-notes/2021-okta-identity-engine/index.md [hunk 6]
```diff
@@ -4,6 +4,18 @@ title: Okta Identity Engine API Products Release Notes 2021
 <ApiLifecycle access="ie" /><br>
 <ApiLifecycle access="Limited GA" /><br>
 
+## October
+
+### Monthly Release 2021.10.0
+
+| Change                                                                   | Expected in Preview Orgs |
+|--------------------------------------------------------------------------|--------------------------|
+| [Brands API option to remove sign-in page footer message](#brands-api-option-to-remove-sign-in-page-footer-message) | October 6, 2021 |
```
Comment (vanngo-okta) [inline packages/@okta/vuepress-site/docs/release-notes/2021-okta-identity-engine/index.md:13]:
```suggestion
| [New Brands API option to remove sign-in page footer message](#new-brands-api-option-to-remove-sign-in-page-footer-message) | October 6, 2021 |
```

File: packages/@okta/vuepress-site/docs/release-notes/2021/index.md [hunk 7]
```diff
@@ -1,6 +1,39 @@
...
+
+You can now configure your org to use the Native Single-Sign On (SSO) flow. The [Native SSO](/docs/guides/configure-native-sso/) feature allows you to protect native OpenID Connect applications and achieve SSO and Single Logout between native applications. <!--428053-->
+
+#### New Brands API option to remove sign-in page footer message
+
+You can now remove "Powered by Okta" and "© 2021" from the Okta-hosted sign-in page using either the [Customizations > Footer](https://help.okta.com/okta_help.htm?id=ext-custom-footer) option in the Admin Console or the [Brands API](/docs/reference/api/brands/#brand-api-objects). <!--OKTA-424736-->
+
+#### Bug fixed in 2021.10.0
+
+Read-Only Admins were unable to [list key credentials for an application](/docs/reference/api/apps/#list-key-credentials-for-application) from the Apps API. (OKTA-430970)
```
Comment (vanngo-okta) [inline packages/@okta/vuepress-site/docs/release-notes/2021/index.md:35]:
```suggestion
Read-only admins were unable to [list key credentials for an application](/docs/reference/api/apps/#list-key-credentials-for-application) from the Apps API. (OKTA-430970)
```

File: packages/@okta/vuepress-site/docs/release-notes/2021/index.md [hunk 8]
```diff
@@ -1,6 +1,39 @@
...
+
+To secure SAML assertions from attacks and to adopt a stronger security mechanism, Okta now supports AES128-GCM and AES256-GCM encryptions modes in addition to AES-128 and AES-256 for SAML applications. <!--OKTA-428056-->
+
+#### Authentication API returns UD User Profile locale is GA in Preview
+
+The response for the primary Authentication API includes the [User Profile's locale](/docs/reference/api/authn/#user-profile-object) value that is the same as the User Profile locale value in Universal Directory (UD). <!--OKTA-430697-->
+
+#### Native SSO support is GA in Preview
+
+You can now configure your org to use the Native Single-Sign On (SSO) flow. The [Native SSO](/docs/guides/configure-native-sso/) feature allows you to protect native OpenID Connect applications and achieve SSO and Single Logout between native applications. <!--428053-->
```
Comment (vanngo-okta) [inline packages/@okta/vuepress-site/docs/release-notes/2021/index.md:27]:
```suggestion
You can now configure your org to use the Native SSO flow. The [Native SSO](/docs/guides/configure-native-sso/) feature allows you to protect native OpenID Connect applications and achieve SSO and Single Logout between native applications. <!--428053-->
```

Comment (vanngo-okta) [reply packages/@okta/vuepress-site/docs/release-notes/2021/index.md:27]: Graham says SSO is in the glossary. No need to spell it out.

**Pull request merged**